sdk/core/core-http/lib/credentials/serviceBusSASServiceClientCredentials.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { HttpHeaders } from "../httpHeaders";
+import { WebResource } from "../webResource";
+import { ServiceClientCredentials } from "./serviceClientCredentials";
+import { Constants } from "../util/constants";
+import { objectIsNull } from "../util/utils";
+import crypto from "crypto";
+import util from "util";
+
+const HeaderConstants = Constants.HeaderConstants;
+
+export class ServiceBusSASServiceClientCredentials implements ServiceClientCredentials {
```

Comment:
Since this credential is specific to a particular Azure service, it should live in the library for that service and not core-http.
Please move this to the Service Bus library.